platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDbStatistics.java

```diff
@@ -229,37 +238,48 @@ public void registerMetrics(final Metrics metrics) {
 
         // Compaction
 
-        for (int i = 0; i < config.maxCompactionLevel(); i++) {
-            int level = i + 1;
+        for (int i = 0; i <= config.maxCompactionLevel(); i++) {
```

Comment:
nit: The bulk of the loop might read a little better if we renamed `i` to `level`.

Reply:
Fair enough. I'll rename it

Reply:
Done